OrbitGl/CaptureSerializer.cpp

```diff
@@ -162,7 +160,7 @@ void CaptureSerializer::Load(const std::wstring& a_FileName) {
 
     // Timers
     Timer timer;
-    while (file.read((char*)&timer, sizeof(Timer))) {
+    while (file.read(reinterpret_cast<char*>(&timer), sizeof(Timer))) {
```

Comment:
I suspect this is windows-only and this is why clang did not complain about it.

Reply:
Yeah this code is ifdefed out on Linux, it should be revived btw so that we can load serialized captures on Linux also.

Reply:
Exactly, I was having a look as this code as I'm looking into saving and loading.